packages/website/pages/bids.tsx

```diff
@@ -35,17 +39,17 @@ const Bids = () => {
   const setIndexLoader = useSetRecoilState(indexLoaderState);
   const setAlertMessage = useSetRecoilState(alertMessageState);
 
-  // const { loading, data, error } = useQuery(GET_BIDS, {
-  //   variables: {
-  //     bidder: accountId || "",
-  //   },
-  // });
+  const { loading, data, error } = useQuery(GET_BIDS, {
```

Comment:
is this a temporary fix?

Reply:
Nope. These lines are there because subgraph had a failure in their service. So we had to constantly change between subgraph and the contract to fetch bid data when the subgraph is not working. Since subgraph service is stable now, we can remove the contract code from here. :)

Reply:
Got it, thanks!